appsdk/sdk.go

```diff
@@ -124,6 +124,11 @@ func (sdk *AppFunctionsSDK) LoadConfigurablePipeline() ([]appcontext.AppFunction
 
 	sdk.usingConfigurablePipeline = true
 
+	sdk.TargetType = nil
```

Comment:
Add a comment that nil will default to EdgeX Event

Reply:
Added comment in sdk.go